### ISSUE TITLE
Tooltip content is not re-aligned when the content changes

### ISSUE THREAD

**Issue by jerolimov** [patternfly/patternfly-react#6589]
**Describe the issue. What is the expected and unexpected behavior?**
If the `<Tooltip content={} ... />` changes while the tooltip is opened, the allow and the tooltip position is not automatically realigned. The easiest case to see this is using the `ClipboardCopy` component with a different `hoverTip` and `clickTip` text.

Take a look into the examples at https://www.patternfly.org/v4/components/clipboard-copy/

For example:

```
<ClipboardCopy hoverTip="Copy" clickTip="Copied">This is editable</ClipboardCopy>
```

Demo (Movie which shows that the tooltip jumps after the user has pressed the ClipboardCopy button):

![missaligned-copy-tooltip](https://user-images.githubusercontent.com/139310/141470788-6aa61203-0990-4f43-a16a-1fd3e9690dd9.gif)

But this happen also for other components which updates the tooltip content.

**Please provide the steps to reproduce. Feel free to link CodeSandbox or another tool.**
https://codesandbox.io/s/patternfly-tooltip-wrong-position-after-content-change-koyo3

**Is this a bug or enhancement? If this issue is a bug, is this issue blocking you or is there a work-around?**
Something between bug and enhancement... :)

Just a small UI glitch, so it doesn't block us, but it also doesn't look good in our OpenShift developer console:

When coping a Secret:
![copying-secret](https://user-images.githubusercontent.com/139310/141471587-8d2246fe-b377-4bbc-a4be-f1516bdca9e8.gif)

After importing a Deployment (upcoming feature):
![copy-after-import](https://user-images.githubusercontent.com/139310/141472108-8e319ed5-ba35-4b8c-8295-5544be3b48ed.gif)

**What is your product and what release version are you targeting?**
OpenShift developer console

**Comment by christianvogt** [comment]
Duplicate of https://github.com/patternfly/patternfly-react/issues/5620

**Comment by tlabaj** [comment]
This is resolved with PR #6598
